packages/esm-ward-app/src/ward-patient-card/card-rows/mother-child-row.extension.tsx

```diff
@@ -0,0 +1,110 @@
+import { InlineNotification } from '@carbon/react';
+import { BabyIcon, MotherIcon } from '@openmrs/esm-framework';
+import classNames from 'classnames';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { type MothersAndChildrenSearchCriteria, useMotherAndChildren } from '../../hooks/useMotherAndChildren';
+import { type WardPatientCard } from '../../types';
+import WardPatientSkeletonText from '../row-elements/ward-pateint-skeleton-text';
+import WardPatientAge from '../row-elements/ward-patient-age';
+import WardPatientIdentifier from '../row-elements/ward-patient-identifier';
+import WardPatientLocation from '../row-elements/ward-patient-location';
+import WardPatientName from '../row-elements/ward-patient-name';
+import wardPatientCardStyles from '../ward-patient-card.scss';
+import styles from './mother-child-row.scss';
+
+const motherAndChildrenRep =
+  'custom:(childAdmission,mother:(person,identifiers:full,uuid),child:(person,identifiers:full,uuid),motherAdmission)';
+
+/**
+ * This extension displays the mother or children of the patient in the patient card.
+ *
+ * @param param0
+ * @returns
+ */
+const MotherChildRowExtension: WardPatientCard = ({ patient }) => {
+  const { t } = useTranslation();
+
+  const getChildrenRequestParams: MothersAndChildrenSearchCriteria = {
+    mothers: [patient.uuid],
+    requireMotherHasActiveVisit: true,
+    requireChildHasActiveVisit: true,
+    requireChildBornDuringMothersActiveVisit: true,
+  };
+
+  const getMotherRequestParams: MothersAndChildrenSearchCriteria = {
+    children: [patient.uuid],
+    requireMotherHasActiveVisit: true,
+    requireChildHasActiveVisit: true,
+    requireChildBornDuringMothersActiveVisit: true,
+  };
+
+  const {
+    data: childrenData,
+    isLoading: isLoadingChildrenData,
+    error: childrenDataError,
+  } = useMotherAndChildren(getChildrenRequestParams, true, motherAndChildrenRep);
+  const {
+    data: motherData,
+    isLoading: isLoadingMotherData,
+    error: motherDataError,
+  } = useMotherAndChildren(getMotherRequestParams, true, motherAndChildrenRep);
+
```

Comment:
I see here why it's annoying to make two queries... :)  we can continue to work on revamping this.